src/java.base/share/classes/java/io/ObjectInputFilter.java

```diff
@@ -523,6 +523,8 @@ static ObjectInputFilter rejectUndecidedClass(ObjectInputFilter filter) {
      * {@systemProperty jdk.serialFilter}, its value is used to configure the filter.
      * If the system property is not defined, and the {@link java.security.Security} property
      * {@code jdk.serialFilter} is defined then it is used to configure the filter.
+     * The filter is created as if {@link #createFilter(String) createFilter} is called;
+     * if the filter string is invalid, an {@link ExceptionInInitializerError} is thrown.
      * Otherwise, the filter is not configured during initialization and
```

Comment:
Hello Roger, the new line looks good to me. However, with this new line now staying between the "If the Java virtual machine ..." and the "Otherwise, ..." lines,  I had to re-read this "Otherwise, the filter is not ..." line twice to see which "if" it corresponds to, because there's a "if the filter string is invalid" on the previous line.
Do you think these lines might have to be rearranged?

Reply:
Good suggestion, I'll take that up in a new issue related to incomplete initialization.